src/applets/night-light/indicator_window.ui

```diff
@@ -0,0 +1,169 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!-- Generated with glade 3.20.0 -->
+<interface>
+  <requires lib="gtk+" version="3.20"/>
+  <object class="GtkAdjustment" id="adjustment">
+    <property name="upper">6000</property>
```

Comment:
Not defined. I tested the value and it was around the limit of my monitor so I guess it's gonna be different depending on the monitor?
Should I just remove the limit?